src/components/AddUsersModal/index.jsx

```diff
@@ -102,10 +102,27 @@ class AddUsersModal extends React.Component {
       greeting: formData['email-template-greeting'],
       closing: formData['email-template-closing'],
     };
-    const hasTextAreaEmails = !!formData['email-addresses'];
-    const emails = hasTextAreaEmails ? formData['email-addresses'].split(/\r\n|\n/) : formData['csv-email-addresses'];
-    const { validEmails } = validateEmailAddresses(emails);
-    options.user_emails = validEmails;
+
+    // Validate email addresses from both the text area and the CSV file and
+    //   submit to the backend for emails to be sent and/or error to be displayed
+    const emails = [];
+    if (formData['email-addresses'] && formData['email-addresses'].length) {
+      const validatedTextAreaEmailAddressData = validateEmailAddresses((
```

Comment:
If we concat all the emails (valid or otherwise) into the `emails` array, we could call `validateEmailAddresses` just once. See https://github.com/edx/frontend-app-admin-portal/pull/316/files#r463620542.